src/PrestaShopBundle/Kernel/ModuleRepository.php

```diff
@@ -67,7 +74,12 @@ public function __construct(Connection $connection, $databasePrefix)
     public function getActiveModules()
     {
         if (null === $this->activeModules) {
-            $sth = $this->connection->query('SELECT name FROM ' . $this->tableName . ' WHERE active = 1');
+            $id_shops = $this->context->shop->getContextListShopID();
+
+            $sth = $this->connection->query('SELECT m.`name`
```

Comment:
Could we improve this variable name? `$sth` is really not helpful when you try to understand what's in this query

Reply:
changed 😊